pkg/pipelinerunmetrics/metrics.go

```diff
@@ -0,0 +1,138 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pipelinerunmetrics
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"knative.dev/pkg/logging"
+	"knative.dev/pkg/metrics"
+)
+
+var (
+	sgCount = stats.Float64("signed_total",
+		"Total number of signed pipelineruns",
+		stats.UnitDimensionless)
+
+	sgCountView *view.View
+
+	plCount = stats.Float64("payload_stored_total",
```

Comment:
We can have one more metrics. `Create_Signed_Total`.

Reply:
Created  4 metrics as discussed